package Gather;

enum GtMessageType {
    GT_LOGIN_REQUEST = 1;
    GT_LOGIN_RESPONSE = 2;
    GT_LOGOUT_MESSAGE = 3;
}

message GtSimpleMessage {
    optional int32 data1 = 1;
    optional int32 data2 = 2;
    optional string data3 = 3;
    optional string data4 = 4;
}

message GtLoginRequest {
    optional string user = 1;
    optional string passwd = 2;
}
